resources/teamcity/android/android-actions.inc.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Keyman is copyright (C) SIL Global. MIT License.
+
+android_clean_action() {
+  builder_echo start "clean" "Cleaning artifact directories"
+  # shellcheck disable=SC2154
+  "${KEYMAN_ROOT}/android/build.sh" clean
+  builder_echo end "clean" success "Finished cleaning artifact directories"
+}
+
+android_build_action() {
+  builder_echo start "build" "Building Keyman for Android"
+  "${KEYMAN_ROOT}/android/build.sh" configure build,test:${TARGETS} --debug --ci
```

Comment:
What is `${TARGETS}` here? Can we pass it as a parameter instead?

And is it deliberate that we configure all targets but only build,test $TARGETS?

Reply:
> Is it deliberate that we configure all targets but only build,test $TARGETS?

I don't know. That's the way it's currently done on TC.

> What is ${TARGETS} here? Can we pass it as a parameter instead?

Done.